3c.segment_cells/scripts/5.endpoint_manual_alignment.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import argparse
+import pathlib
+import sys
+
+import cv2
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import scipy.ndimage as scind
+import scipy.sparse
+from centrosome.filter import stretch
+from scipy.fftpack import fft2, ifft2
```

Comment:
Double checking: are all these imports needed for this notebook? Some of these seemed like the CP function related imports.